### ISSUE TITLE
Question about the implementation of LwF

### ISSUE THREAD

**Issue by gqk** [GT-RIPL/AlwaysBeDreaming-DFCIL#6]
Nice work!

I have a question about your implementation of LwF, it seems that there is a big gap between the data of LwF reported in your paper and the previous works (iCaRL and End-to-End Incremental Learning).

I also implement the LwF.MC introduced by iCaRL, and my results on CIFAR100 consistent with the data reported by iCaRL, there are also some other implementations, such as [PODNet](https://github.com/arthurdouillard/incremental_learning.pytorch/blob/master/inclearn/models/lwf.py), it seems that PODNet uses the BCELoss, while I use the CrossEntropyLoss instead.

I read your code, the problem may lie in the loss in [LwF.MC](https://github.com/GT-RIPL/AlwaysBeDreaming-DFCIL/blob/main/learners/kd.py#L217).

**Comment by gqk** [comment]
Sorry, I wrongly think the data reported in the paper belongs to LwF.MC. In addition, I found that [ft classification](https://github.com/GT-RIPL/AlwaysBeDreaming-DFCIL/blob/main/learners/datafree.py#L327)  is actually performed on all 100 classes, which reduces the last performance from 45% to 42% (10000 inversion iterations).
